integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala

```diff
@@ -141,11 +138,7 @@ class CarbonStrategies(sqlContext: SQLContext) extends QueryPlanner[SparkPlan] {
         predicates,
         useUnsafeCoversion = false)(sqlContext)
       projectExprsNeedToDecode.addAll(scan.attributesNeedToDecode)
-      val updatedAttrs = scan.attributesRaw.map(attr =>
```

Comment:
why this code is deleted?
please create another PR if logic changed is required, and keep this PR change minimal